go/libkb/kbsig.go

```diff
@@ -614,3 +612,35 @@ type SigMultiItem struct {
 	SigInner   string `json:"sig_inner"`
 	TeamID     string `json:"team_id"`
 }
+
+// PerUserKeyProof creates a proof introducing a new per-user-key generation.
+// `signingKey` is the key signing in this new key. Not to be confused with the derived per-user-key signing key.
+// `reverseSig` can be nil. A nil `reverseSig` can be used to produce the inner of the reverse sig.
+func PerUserKeyProof(me *User, pukSigKID keybase1.KID, pukEncKID keybase1.KID, generation keybase1.PerUserKeyGeneration,
+	signingKey GenericKey, reverseSig *string) (*jsonw.Wrapper, error) {
```

Comment:
Maybe split the args here up into their own lines? I don't know if we've been consistent about this thought. I'm kind of surprised gofmt doesn't have a strong opinion about it.